src/controls/controls.tsx

```diff
@@ -13,211 +13,211 @@ import { JogButtons } from "./jog_buttons";
 import { AxisInputBoxGroup } from "./axis_input_box_group";
 
 export class StepSizeSelector extends React.Component<any, any> {
-    cssForIndex(num: number) {
-        let choices = this.props.choices;
-        let css = "move-amount no-radius ";
-        if (num === _.first(choices)) {
-            css += "leftmost ";
-        }
-        if (num === _.last(choices)) {
-            css += "rightmost ";
-        }
-        if (num === this.props.selected) {
-            css += "move-amount-selected ";
-        }
-        return css;
+  cssForIndex(num: number) {
+    let choices = this.props.choices;
+    let css = "move-amount no-radius ";
+    if (num === _.first(choices)) {
+      css += "leftmost ";
     }
-
-    render() {
-        return (<div className="move-amount-wrapper">
-            {
-                this.props.choices.map(
-                    (item: number, inx: number) => <button
-                        className={this.cssForIndex(item)}
-                        onClick={() => this.props.selector(item)}
-                        key={inx} >{item}</button>
-                )
-            }
-        </div>);
+    if (num === _.last(choices)) {
+      css += "rightmost ";
+    }
+    if (num === this.props.selected) {
+      css += "move-amount-selected ";
     }
+    return css;
+  }
+
+  render() {
+    return (<div className="move-amount-wrapper">
+      {
+        this.props.choices.map(
+          (item: number, inx: number) => <button
+            className={this.cssForIndex(item)}
+            onClick={() => this.props.selector(item)}
+            key={inx} >{item}</button>
+        )
+      }
+    </div>);
+  }
 }
 
 const showUrl = (url: string, dirty: boolean) => {
-    if (dirty) {
-        return <p>Press save to view.</p>;
-    } else {
-        if (url.indexOf("/webcam_url_not_set.jpeg") !== -1) {
-            return <div className="webcam-stream-unavailable">
-                <img src={url} />
-                <text>Camera stream not available.
+  if (dirty) {
+    return <p>Press save to view.</p>;
+  } else {
+    if (url.indexOf("placeholder_farmbot") !== -1) {
```

Comment:
We can use `.includes()` on these now. I don't think that was the case when we wrote this the first time around.